sunbeam-python/sunbeam/core/openstack_api.py

```diff
@@ -43,16 +43,17 @@ def get_admin_connection(jhelper: JujuHelper) -> openstack.connection.Connection
 
 
 def guests_on_hypervisor(
-    hypervisor_name: str, jhelper: JujuHelper
+    hypervisor_name: str, conn: openstack.connection.Connection
```

Comment:
This change seems irrelevant, do you really need it? If yes, please also update the docstring

Reply:
Updated docstring.